src/main/groovy/projet/gl51/Store/MemoryProductStorage.groovy

```diff
@@ -0,0 +1,44 @@
+package projet.gl51.Store
+
+class MemoryProductStorage implements ProductStorage {
+    List<Product> productList = []
```

Comment:
Une liste ??? il vaut mieux utiliser une map plutot !
-> tu aurais une complexité en o(1) au lieu de o(n) dans ton cas...
http://javabypatel.blogspot.com/2015/10/time-complexity-of-hashmap-get-and-put-operation.html